Урок 6. Added 5 new tests:

VickyAQA opened the pull request:
1) Place a new order for a pet
2) Get order information by ID
3) Delete purchase order by ID
4) Get information about a non-existent order
5) Returns pet inventories by status

Test Run:
http://5.181.109.28:8080/job/VP_PetShopApiTests/7/allure/#behaviors/4ed64c53877284ee9eb43df4a97e6332

VickyQAkid committed:
Added 5 new tests:
1) Place a new order for a pet
2) Get order information by ID
3) Delete purchase order by ID
4) Get information about a non-existent order
5) Returns pet inventories by status

dmitryvshivtsev commented on tests/test_store.py:
```diff
@@ -0,0 +1,78 @@
+
+        with allure.step("Check status code"):
+            assert response.status_code == 200, "The response code did not match the expected one"
+            jsonschema.validate(response_json, STORE_SCHEMA)
```
Это скорее intentory_schema, а не store.

dmitryvshivtsev commented on tests/schemas/store_schema.py:
```diff
@@ -0,0 +1,13 @@
+            "type": "integer"
+        }
+    },
+    "required": ["approved","delivered"],
```
Есть ещё статус `placed`. Добавь его в схему, как один из возможных

VickyQAkid committed:
Added 5 new tests:
1) Place a new order for a pet
2) Get order information by ID
3) Delete purchase order by ID
4) Get information about a non-existent order
5) Returns pet inventories by status
